### ISSUE TITLE
No accessible names - Our Story, Our People, Our Work boxes/frames

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#72]
This element has no HTML markup that would provide an accessible name for the element. A screen reader would either read nothing for this element or may make a guess.
